.github/workflows/pytests.yml

```diff
@@ -0,0 +1,46 @@
+name: Unit and Integration tests with pytest
+
+on:
+  pull_request:
+    types: [ assigned, opened, synchronize, reopened ]
+#    paths: [ 'services/py_api/**' ]
+  push:
+    branches:
+      - main
+
+defaults: 
+  run: 
+    working-directory: services/py_api/
+
+jobs:
+  run_unit_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ "3.12.6" ]
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
```

Comment:
Should probably cache those

Reply:
@asynchroza Fixed in 29afd142bb53998e72a12beac782052f46646bcf